lib/fastlane/plugin/docker/actions/docker_compose.rb

```diff
@@ -0,0 +1,61 @@
+module Fastlane
+  module Actions
+    class DockerComposeAction < Action
+      def self.run(params)
+        UI.message "running docker-compose ..."
+
+        unless `which docker-compose`.include?("docker-compose")
+          UI.error("docker-compose is not found")
+          UI.error("please read the following https://docs.docker.com/compose/install/ on how to install it.")
+          UI.user_error!("docker-compose not installed")
+        end
+
+        command = "docker-compose "
+        command << params[:files].map { |item| " -f " + item }.join + " " if params[:files]
+        command << "--project-name='{params[:project].shellescape}' " if params[:project]
+        command << params[:command]
+        command << " -d " if params[:command] == "up" && params[:detach]
+
+        Actions.sh command, log: true
+      end
+
+      def self.description
+        "Build docker images in the current directory"
```

Comment:
I haven't used docker-compose too much, so I might be wrong here, but it doesn't just build images, right?